acceptance/options/puppetserver_embedded_db.rb

```diff
@@ -0,0 +1,12 @@
+# We are eval'd in the scope of the acceptance framework's option-parsing
```

Comment:
@kbarber , yes we will be adding a postgresql variant once the embedded case has been vetted in CI.